src/step-console-index.js

```diff
@@ -1,7 +1,56 @@
-/**
- * step 1의 시작점이 되는 파일입니다.
- * 브라우저 환경에서 사용하는 css 파일 등을 불러올 경우 정상적으로 빌드할 수 없습니다.
- */
+import Lotto from './tests/Lotto';
+import Winning from './tests/Winning';
+import { WinningRank } from './tests/constant';
 
+const readline = require('readline');
 
-console.log("Hello, World!");
+function readLineAsync(query) {
+	return new Promise(resolve => {
+		const rl = readline.createInterface({
+			input: process.stdin,
+			output: process.stdout
+		});
+
+		rl.question(query, input => {
+			rl.close();
+			resolve(input);
+		});
+	});
+}
+
+async function play() {
+	const lotto = new Lotto();
+
+	const purchaseAmount = await readLineAsync('구입금액을 입력해 주세요. > ');
+	const lottoNumbers = lotto.generatePurchasedLottoNumbers(purchaseAmount);
+
+	const quantity = lottoNumbers.length;
+
+	console.log(`${quantity}개를 구매했습니다.`);
+
+	for (let i = 0; i < quantity; i++) {
+		console.log(`[${lottoNumbers[i].join(', ')}]`);
+	}
+
+	const winningNumbersInput = await readLineAsync('당첨 번호를 입력해 주세요.');
+	const winningNumbers = winningNumbersInput.split(',').map(Number);
+
+	const bonusNumberInput = await readLineAsync('보너스 번호를 입력해 주세요.');
+	const bonusNumber = Number(bonusNumberInput);
+
+	const { prizeCounts, totalPrize } = Winning.calculateResults(lottoNumbers, winningNumbers, bonusNumber);
+	const winningRate = Winning.calculateRate(totalPrize, purchaseAmount);
```

Comment:
```suggestion
	const { prizeCounts, totalPrize } = Winning.calculateResults("이상한 값", "이상한 로또 우승자", "이상한 보너스 번호");
	const winningRate = Winning.calculateRate(totalPrize, purchaseAmount);
```
인자를 이렇게 넘겨주면 무슨일이 일어날까요~?